app/src/main/java/amazingme/controller/ActivityManager.java

```diff
@@ -19,13 +19,19 @@ public static ActivityManager getInstance() {
     }
 
     public void goTo(final Activity context, final EnumeratedActivity endingActivity) {
-        context.startActivity(new Intent(context, endingActivity.getAppCompatActivity()));
+        if (endingActivity.isActiveGame()) {
+            this.goTo(context, endingActivity.getAppCompatGame());
+        } else {
+            this.goTo(context, endingActivity.getAppCompatActivity());
+        }
     }
 
     public void goTo(final Activity context, final Class<? extends AppCompatActivity> endingActivity) {
         context.startActivity(new Intent(context, endingActivity));
     }
 
+
+    // TODO i think we can get rid of this?? in the review, remind me if you know it's used anywhere
```

Comment:
I don't personally use it. I would just check usages and if theres no references means we can get rid of it.